File: miscellaneous/Divisible_Sum.java
import java.util.*;

public class Main {
  public static boolean valid(List<Integer>arr,int val){
    int n=arr.size();
    int flag=1;
    for(int i=0;i<n;i++){
   if(val%arr.get(i)==0){ 
    flag=0;
    return false;
   }
    
    }
    if(flag==1)return true;
    return false;
}
    public static void main(String[] args) {
   
     Scanner sc=new Scanner(System.in);
   System.out.println("Enter dimension : ");
    int n=sc.nextInt();
    List<Integer>arr=new ArrayList<>();
   System.out.println("Enter the elements of the array : ");
    for(int i=0;i<n;i++){
        int item=sc.nextInt();
        
        arr.add(item);
    }
    int k=sc.nextInt();
    int sum=0;
   System.out.println("Enter K : ");
    
    int i=k;
    int j=1;
    while(i>0){
    if(valid(arr,j)){
       System.out.println(j);
        sum+=j;
        i--;
    }
    j++;
    }
   System.out.println("\nSum : "+sum);

  }
  
}

File: miscellaneous/Divisible_sum.cpp
# include<bits/stdc++.h>
using namespace std;
bool valid(vector<int>&arr,int val){
    int n=arr.size();
    int flag=1;
    for(int i=0;i<n;i++){
   if(val%arr[i]==0){ 
    flag=0;
    return false;
   }
    
    }
    if(flag==1)return true;
}
int main(){
    int n;
    cout<<"Enter dimension : ";
    cin>>n;
    vector<int>arr;
    cout<<"Enter the elements of the array : ";
    for(int i=0;i<n;i++){
        int item;
        cin>>item;
        arr.push_back(item);
    }
    int k;
    int sum=0;
    cout<<"Enter K : ";
    cin>>k;
    int i=k;
    int j=1;
    while(i){
    if(valid(arr,j)){
        cout<<j<<" ";
        sum+=j;
        i--;
    }
    j++;
    }
    cout<<"\nSum : "<<sum;

}
